src/NetMQ/Monitoring/INetMQMonitor.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Threading.Tasks;
+
+namespace NetMQ.Monitoring
+{
+    /// <summary>
+    /// NetMQMonitor interface, implement to fake the NetMQMonitor in tests.
+    /// </summary>
+    public interface INetMQMonitor : IDisposable
```

Comment:
Comments have been copied to this interface from `NetMQMonitor`. Where they're identical, they can now be removed from the implementation as they'll be inherited implicitly.